auth-web/src/views/auth/staff/ReviewAccountView.vue

```diff
@@ -317,7 +333,8 @@ export default class ReviewAccountView extends Vue {
     await DocumentService.getSignedAffidavit(this.accountUnderReviewAffidavitInfo?.documentUrl, `${this.accountUnderReview.name}-affidavit`)
   }
 
-  private openModal (isRejectModal:boolean = false, isConfirmationModal: boolean = false, rejectConfirmationModal:boolean = false) {
+  private openModal (isRejectModal:boolean = false, isConfirmationModal: boolean = false, rejectConfirmationModal:boolean = false, isMoveToPendingModal: boolean = false) {
```

Comment:
could be a candidate for an object as a parameter, 4 is ok I think though

Reply:
it should also pick up the boolean typing automatically since you're setting the default